apollo-router/Cargo.toml

```diff
@@ -52,6 +52,8 @@ docs_rs = ["router-bridge/docs_rs"]
 # and not yet ready for production use.
 telemetry_next = []
 
+experimental_hyper_fork = []
```

Comment:
Could this have a more descriptive name like `experimental_hyper_header_limits`?